### ISSUE TITLE
Stepping into new/malloc in TTD freezes windbg

### ISSUE THREAD

**Issue by mattkse** [microsoftfeedback/WinDbg-Feedback#161]
I was using windbg on a TTD recorded trace and stepped into the new/malloc of an object and the debugger became unresponsive to any commands -- cannot even click "Stop Debugging".

**Comment by mattkse** [comment]
I just realized there was a background pop-up asking for credentials after I closed windbg.

**Comment by lzybkr** [comment]
Do you know which version of the debugger you were using? This was a known issue in the past but we _think_ we've addressed those issues in recent debuggers.

**Comment by mattkse** [comment]
It was version 1.2402.24001.0.

**Comment by mattkse** [comment]
I think it was just waiting for the background pop-up to disappear.

**Comment by lzybkr** [comment]
That is actually the latest version. Do you recall which program was prompting for authentication?

**Comment by mattkse** [comment]
No, it was just a pop-up in the background.  I've actually reproduced this behavior a few times, it should be easy to try to reproduce.

**Comment by KenSykes** [comment]
We are cleaning up the issues list. This bug is older than six months. If you still see this issue in the current debugger and would like to see a fix please submit a new report.\ Thanks!
